src/Psalm/ErrorBaseline.php

```diff
@@ -104,8 +104,11 @@ public static function read(FileProvider $fileProvider, string $baselineFile): a
 
             $files[$fileName] = [];
 
-            /** @var \DOMElement $issue */
             foreach ($file->childNodes as $issue) {
+                if (!$issue instanceof \DOMElement) {
```

Comment:
I hesitated on using `/** @var \DOMNode|\DOMComment $issue */` instead, but I thought as both have `DOMNode` as an ancestor would be better to check that we iterate only on `DOMElement` node.

I can apply that instead if you'd rather do this.

Reply:
As long as everything still works, I don't really care :)